### ISSUE TITLE
Game went to black and white after first launch

### ISSUE THREAD

**Issue by zeziba** [DarkPacks/SkyFactory-5#525]
### Bug Description

After launching the game the first time and getting past the accessibility initial prompt the game when completely black and white.

This happens right after it finished it's initial load now as well and a restart does not fix it.

### Did this behavior used to work in the previous version?

Unknown

### Steps to Reproduce (for bugs)

1. Installed modpack
2. Launch game
3. Do accessibility stuff
4. Get a fun black and white experience.

### Client/Server Log:

https://gist.github.com/zeziba/0dfdebdf29941d5a17e7571dbb680b7f

### Modpack Version world created in:

5.0.8

### Additional Content Installed?

n/a

### Modpack Version:

5.0.8

### Java Version:

17.0.2 2022-01-18

### Launcher Used:

prisim, curseforge

### Memory Allocated:

9000

### Single Player/LAN/Server:

Singleplayer

### Optifine Installed:

No

### Shaders Enabled:

No

**Comment by Marioluigi7896** [comment]
The Black and White Screen is an intended effect, as when playing the game you unlock individual colors and will get those colors to appear in game even in the menus, the accessibility screen text is customized to let you know of this visual effect. 
The menu keeps the effect of the last world you loaded, so if you only have 1 color unlocked you will only see that color on the menu, which is why a restart does not fix this. To see the fully colored menu you need to enter a world where you have unlocked all colors.
